### DISCUSSION TITLE
Found conflicts between different versions of "Magick.NET-Q8-AnyCPU" that could not be resolved

### DISCUSSION THREAD

**Discussion by kchinmay07** [dlemstra/Magick.NET#1677] [Help]
**MY CODE FOR HORIZONTAL  SCREENSHOT** 

1.     ITakesScreenshot ts = (ITakesScreenshot)driver;
2.     Screenshot ss = ts.GetScreenshot();
3.     String path = System.Reflection.Assembly.GetCallingAssembly().CodeBase;
4.     String finalpath = pth.Substring(0, pth.LastIndexOf("bin")) + "REPORTING\\FailedSS\\" + SRNName + ".png";
5.     String localPath = new Uri(finalpth).LocalPath;
6.     //screenshot.SaveAsFile(localpath,ScreenshotImageFormat.Png); I replace below 2 line code for full page screenshot
7.     VerticalCombineDecorator vcd = new VerticalCombineDecorator(new ScreenshotMaker());
8.     driver.TakeScreenshot(vcd).ToMagickImage().ToBitmap().save(localPth);
9.     return localPth;
i commented line number 6 , and added 7 and 8 to get VERTICAL SCREENSHOT /VCD - (iam following this youtube channel- https://www.youtube.com/watch?v=MUVCE550yw8)
and my WDSE(Version -0.1.5.4)  AND Magick.NET-Q8-AnyCPU (13.9.0)
i am getting red line below the 8th line till ToMagicimage() and .ToBitmap() is not coming and vertical screenshot not getting generated
 when i run the test , i am getting below error 
"Found conflicts between different versions of "Magick.NET-Q8-AnyCPU" that could not be resolved."
Can someone please help me . Thanks in advance

**Comment by dlemstra** [comment]
I suspect that you are mixing 32 and 64 bit versions of this library.

**Reply by kchinmay07** [reply]
![Magick net](https://github.com/user-attachments/assets/8661cc2a-2706-411d-af43-1a1841cdf003)
my pc System type  is  64 bit operating system ,X64 based processor 
WDSE(Version -0.1.5.4) AND Magick.NET-Q8-AnyCPU (13.9.0).

Right now i am getting Horizontal screenshot

**can you send me the pic of  magick.NET  version + WDSE = Which will give  .ToMagickImage().ToBitmap().save(localPth);,** 

reason why i am asking you the pic is - 
the moment i download "Magick.NET-Q8-AnyCPU" version 7.9.0 , i get ToBitmap().save(localPth), but screenshot is not reflecting (and give error in test explorer) .

the moment i download "Magick.NET-Q8-AnyCPU" version 13 version , i am not getting ToBitmap().save(localPth) (while writing the code)
please help

**Reply by dlemstra** [reply]
This must be something in your local setup and I am sorry but I don't have the time to help you with this. It looks like the library that you are using is using an old version of Magick.NET. And they should be referencing Magick.NET.Core now instead. That project also hasn't been maintained for the last 3 years.

**Comment by kchinmay07** [comment]
Go to solution explorer in Visual studio >> search app.config>> open app.config >>search MAGICK.NET and check whether the version provided in the app.config is equal to provided in the package. in my case  app.config was 13.0.0 and in package it was [7.9.0.0](javascript:void(0);)
